packages/adobe_analytics/android/build.gradle

```diff
@@ -41,4 +41,5 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "com.adobe.marketing.mobile:analytics:1.+"
```

Comment:
I think it's better to work with fixed versions.

Reply:
I agree ; I've updated the PR accordingly 👍